docs/index.rst

```diff
@@ -97,6 +97,22 @@ persisted, and so won't survive after the container is finished.
     via the `Dockerfile`. Commits to master will result in the Dockerfile being
     rebuilt on the docker hub.
 
+Hacking on the Docker image
```

Comment:
Yep, excellent idea! At some point, do we want to have a specific topic/section/file just for docker related stuff?